app/lib/data_type_validator.rb

```diff
@@ -39,7 +39,7 @@ def is_email?(value)
   end
 
   def is_url?(value)
-    value.match?(/^((ftp|http|https):\/\/)?[a-zA-Z0-9]+(.[a-zA-Z0-9])?(:[0-9]+)?/)
+    value.match?(/^((ftp|http|https):\/\/)?[a-zA-Z0-9]+((\.[a-zA-Z0-9]+)|(:[0-9]+)+)/)
```

Comment:
This regex doesn't allow for multiple periods in a url, and it allows more than one port. I'm not sure how big of an issue it is, but `123.0.0.0:3000` would not match, and `localhost:3000:3000` would match